lib/routes/totp.js

```diff
@@ -180,7 +180,7 @@ module.exports = (log, db, customs, config) => {
         },
         validate: {
           payload: {
-            code: isA.string().max(32).required(),
+            code: isA.string().max(32).regex(/^[0-9]+$/).required(),
```

Comment:
nit: Could pull this into the common [validation area](https://github.com/mozilla/fxa-auth-server/blob/aa388cc5eb13333f709030338e07d29efc47deaa/lib/routes/validators.js#L157)